tstime/tstime.go

```diff
@@ -141,3 +142,71 @@ func Parse3339(s string) (time.Time, error) {
 func Parse3339B(b []byte) (time.Time, error) {
 	return parse3339m(mem.B(b))
 }
+
+func Append3339Nano(t time.Time, b []byte) []byte {
```

Comment:
Nit: perhaps AppendRFC3339Nano to make clean it's a specialized form of `time.RFC3339Nano`?